src/App.jsx

```diff
@@ -190,14 +154,23 @@ function App() {
 	if (errorState) {
 		return <ErrorMessage />;
 	}
-
+	function handleChildTabEvent(data) {
+		setData ({...data});
+		let individualStake = parseFloat(data.budget)/16;
+		validators = data.validatorsList.reduce((acc, cur) => {
+			acc.push({name: cur.name, amount: individualStake, avatar: 'default', risk: '0.22'});
```

Comment:
For avatar, you can use the identicon provided by PolkadotJS package. To see how it's used, you can refer to the `ValidatorTable.jsx`component

You can import it using:
```import Identicon from '@polkadot/react-identicon'```

Reply:
@saumyakaran Previously we have used avatar component from chakra, which has beed replaced by '@polkadot/react-identicon'

Reply:
@suyanksaraswat Highly recommended reading: https://docs.google.com/document/d/1AqT8av6Hgbu-gwPxooyFbCZFzOyd2I51VhKP5LGUCkw/edit

> @saumyakaran Previously we have used avatar component from chakra, which has beed replaced by '@polkadot/react-identicon'

Is there any implication or ask here?

Reply:
No. It is just a notification.